devicetypes/sensative/sensative-strips-guard-700.src/sensative-strips-guard-700.groovy

```diff
@@ -0,0 +1,482 @@
+	0x9F: 1 	// Security 2
+]
+
+@Field static int wakeUpIntervalSeconds = 43200
+
+
+metadata {
+	definition (
+		name: "Sensative Strips Guard 700",
+		namespace: "Sensative",
+		author: "Kevin LaFramboise",
+		ocfDeviceType:"x.com.st.d.sensor.contact",
+		mnmn: "SmartThingsCommunity",
+		vid: "6d19b679-a36a-327f-809d-163f8b8d54d9"
+	) {
+		capability "Sensor"
+		capability "Contact Sensor"
+		capability "Tamper Alert"
+		capability "Battery"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Health Check"
+		capability "platemusic11009.firmware"
+
+		attribute "lastCheckIn", "string"
+
+		fingerprint mfr:"019A", prod:"0004", model:"0004", deviceJoinName: "Strips Guard 700"
+	}
+
+	simulator { }
+
+	preferences {
+		configParams.each { param ->
+			input "configParam${param.num}", "enum",
+				title: "${param.name}:",
+				required: false,
+				displayDuringSetup: false,
+				options: param.options
+		}
+
+		input "debugLogging", "enum",
+			title: "Logging:",
+			required: false,
+			defaultValue: 1,
+			options: [0:"Disabled", 1:"Enabled [DEFAULT]"]
+	}
+}
+
+
+def installed() {
+	logDebug "installed()..."
+	state.pendingRefresh = true
+	initialize()
+}
+
+def updated() {
+	if (!isDuplicateCommand(state.lastUpdated, 1000)) {
+		state.lastUpdated = new Date().time
+
+		logDebug "updated()..."
+		
+		initialize()
+
+		if (pendingChanges) {
+			logForceWakeupMessage("The configuration changes will be sent to the device the next time it wakes up.")
+		}
+	}
+}
+
+void initialize() {
+	state.debugLoggingEnabled = (safeToInt(settings?.debugOutput, 1) != 0)
+	
+	if (!device.currentValue("tamper")) {
+		sendEventIfNew("tamper", "clear")
+	}
+	
+	if (!device.currentValue("checkInterval")) {
+		sendEvent(name: "checkInterval", value: ((wakeUpIntervalSeconds * 2) + 300), displayed: falsle, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	}
+}
+
+
+def configure() {
+	logDebug "configure()..."
+
+	state.pendingRefresh = true
+	sendCommands(getConfigureCmds())
+}
+
+List<String> getConfigureCmds() {
+	runIn(6, refreshSyncStatus)
+
+	int changes = pendingChanges
+	if (changes) {
+		log.warn "Syncing ${changes} Change(s)"
+	}
+
+	List<String> cmds = [ ]
+
+	if (state.pendingRefresh) {		
+		cmds << batteryGetCmd()
+		cmds << versionGetCmd()
+		cmds << sensorBinaryGetCmd()
+	}
+	
+	if (state.pendingRefresh || (state.wakeUpInterval != wakeUpIntervalSeconds)) {
+		logDebug "Changing wake up interval to ${wakeUpIntervalSeconds} seconds"
+		cmds << wakeUpIntervalSetCmd(wakeUpIntervalSeconds)
+		cmds << wakeUpIntervalGetCmd()
+	}
+
+	configParams.each {
+		Integer storedVal = getParamStoredValue(it.num)
+		Integer settingVal = getSettingValue(it.num)
+		
+		if ((settingVal != null) && (settingVal != storedVal)) {
+			logDebug "CHANGING ${it.name}(#${it.num}) from ${storedVal} to ${settingVal}"
+			cmds << configSetCmd(it, settingVal)
+			cmds << configGetCmd(it)
+		}
+		else if (state.pendingRefresh) {
```

Comment:
Please remove this line break before the "else if" statement.

Reply:
Removed